dbt/models/default/schema/default.vw_pin_sale.yml

```diff
@@ -76,7 +78,7 @@ models:
             - year
           allowed_duplicates: 2
           config:
-            error_if: ">4032"
+            error_if: ">5032"
```

Comment:
@jeancochrane @wagnerlmichael This test compiles as:

```
select
    pin, year,
    count(*) as num_duplicates
from "z_ci_583_create_a_sales_view_that_combines_iasworld_mydec_and_ccrd_sales_default"."vw_pin_sale"
group by pin, year
having count(*) > 2 + 1
```

Are we sure we only want to look for pin/year combos that show up _more than three times a year_? Wouldn't it make more sense to look for combos that show up more than twice?

Reply:
I think this is a question we should answer empirically: What's the maximum number of sales per PIN per year that appear in our data and are legitimate? And do we have enough confidence in our ability to determine legitimacy such that we can answer the question in the first place?

Reply:
Yeah, I suppose if we're not testing a filter here then it's really a matter of expectations and experience rather than some logic that can be divined. I'm fine with it as, perhaps it's worth an issue of its own for investigation.